src/main.rs

```diff
@@ -114,6 +114,18 @@ fn create_app<'a, 'b>(
                 .default_value(&default_memory)
                 .group("vm-config"),
         )
+        .arg(
+            Arg::with_name("memory-zone")
```

Comment:
One last nitpick, sorry for not commenting on that one earlier: Should we use `--memory-zones` instead of `--memory-zone`?

Reply:
It's really up to you. Based on our current CLI choices, like `--disk`, `--net`, ... we always kept the singular even if we can specify multiple instance of each. Maybe moving everything to plural would make sense at some point, but we could make it as part of a dedicated PR.
Your call :)